src/content/configuration/watch.md

```diff
@@ -146,9 +158,9 @@ On macOS, folders can get corrupted in certain scenarios. See [this article](htt
 
 ### Windows Paths
 
-Because webpack expects absolute paths for many config options such as `__dirname + "/app/folder"` the Windows `\` path separator can break some functionality.
+Because webpack expects absolute paths for many config options such as `__dirname + '/app/folder'` the Windows `\` path separator can break some functionality.
 
-Use the correct separators. I.e. `path.resolve(__dirname, "app/folder")` or `path.join(__dirname, "app", "folder")`.
+Use the correct separators. I.e. `path.resolve(__dirname, 'app/folder')` or `path.join(__dirname, 'app', 'folder')`.
```

Comment:
This might be a naive question, but is there a downside of using double quotes?

Reply:
Hi @hemal7735,

No but we strive for consistency across the docs

Reply:
Thank you @EugeneHlushko . Can you rebase please?

Reply:
For what reason?

Reply:
`This branch is out-of-date with the base branch`